### DISCUSSION TITLE
How to display the route path like Waze

### DISCUSSION THREAD

**Discussion by moisemomo** [mapsforge/vtm#915] [Q&A] [answered]
![waze-guidage-voie_01B0000001669397](https://user-images.githubusercontent.com/53950741/159517649-d4e72e54-b580-4edc-be93-8228178ec360.jpg)

Hi, is it possible to display the line like in the above picture. Having a different color for the path to travel and the path already traveled?
Thanks you.

**Comment by devemux86** [comment, marked as answer]
You can use 2 path layers and update their points while moving.

**Reply by moisemomo** [reply]
Thank you @devemux86 , one of my teammates had the same idea,  i will try that.
